arroba/tests/test_datastore_storage.py

```diff
@@ -390,6 +390,22 @@ def test_create_remote_blob_image_aspect_ratio(self):
         self.assertEqual(21, blob.width)
         self.assertEqual(12, blob.height)
 
+    def test_create_remote_blob_video_aspect_ratio(self):
+        image_bytes = Path(__file__).with_name('video.mp4').read_bytes()
```

Comment:
Looks like `video.mp4` didn't get added to this PR?

Also, minor nit, `video_bytes`